TFExplorer/settings.py

```diff
@@ -125,12 +121,20 @@
 
 USE_TZ = True
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
 
+AUTH_USER_MODEL = 'registration.User'
+
+# JWT Authentication
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
```

Comment:
Please also include `'rest_framework.authentication.SessionAuthentication',` after JWTAuth otherwise admins will also have to sign in using tokens and it will create trouble while accessing the admin panels

Reply:
Added,